CMSgov/design-system
# [WNMGDS-1855] Bugfix - checkbox sizing

## Summary

This work folds in a local style from the Windowshop/Enroll app where small checkbox icons weren't displaying correctly. 

It turned out that app's CSS reseat included a `box-sizing: initial` for all elements, (`*, *::before, *::after { box-sizing: initial; }`) that was breaking the layout of our CSS. They fixed it by adding a local `box-sizing: content-box` to the checkbox styles. I migrated this style into our design system.

While testing this change, I discovered another bug:
![Screen Shot 2022-10-17 at 11 05 43 AM](https://user-images.githubusercontent.com/5159392/196214611-3d9709ad-788a-4dac-851a-009d9082beae.png)

Small checkboxes increase in size on focus. After playing around with the style causing this, I was able to determine we could safely remove the offending CSS. Tested in Chrome, Firefox, and Safari and removing this style had no impact on any checkboxes in our design system.

## How to test

Run Storybook locally and navigate to: http://localhost:6006/?path=/story/components-choicelist--small-option&args=type:checkbox

Clicking new checkboxes (moving focus around) should persist the small sizing.

## Checklist




### If this is a change to code:


### If this is a change to documentation:
